packages/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -26,6 +27,16 @@ export interface MenuProps extends MenuListProps {
    * Wrapper to style and add events for the popup
    */
   menuPopup?: ShorthandProps<React.HTMLAttributes<HTMLElement>>;
+
+  /**
+   * Where the menu is positioned with respect to the trigger
+   */
+  position?: PositioningProps['position'];
+
+  /**
+   * How the menu is aligned wtih respect to the trigger
+   */
+  align?: PositioningProps['align'];
```

Comment:
_off-topic: IMO, we should simply extend there `PositioningProps`, but let's avoid rush with such decisions_

Reply:
it could work, let me try it out

Reply:
I think this decision depends on whether we envision a `Popup` component or not

If we had a `Popup` component then we could encourage consumers with more 'advanced' positioning requirements to use `MenuList` and put that in their own popup ?

Reply:
Resolving this for now, let me build out the submenus in a subsequent PR and check if this is really necessary